jedi.inc

```diff
@@ -553,6 +553,9 @@
   SUPPORTS_FINAL                 Compiler supports final keyword (D9.NET, D10+)
   SUPPORTS_METHODINFO            Compiler supports the METHODINFO directives (D10+)
   SUPPORTS_GENERICS              Compiler supports generic implementations (D11.NET, D12+)
+  SUPPORTS_GENERIC_TYPES         Compiler supports generic implementations of types (D11.NET, D12+, FPC)
+  SUPPORTS_GENERIC_METHODS       Compiler supports generic implementations of methods (D11.NET, D12+, FPC)
+  SUPPORTS_GENERIC_FUNCS         Compilre supports generic implementations of global functions (FPC)
```

Comment:
I would have called this one `SUPPORTS_GENERIC_GLOBAL_METHODS` because I believe it also applies to procedures, not just functions. And adding "global" in the name makes it clearer to me, even if it means it becomes a quite long name

Reply:
I don't agree with the name `GLOBAL_METHODS`. There is no such thing as a "global method" as a method is associated with a class while functions and procedures are not. For global functions or procedures it's either common to use "function" for both (which stems from C, where "procedures" are simply functions returning `void`) or to use "routine". To quote from the Delphi documentation about [Procedures and Functions](http://docwiki.embarcadero.com/RADStudio/Rio/en/Procedures_and_Functions_(Delphi)): 

> Procedures and functions, referred to collectively as routines, are self-contained statement blocks that can be called from different locations in a program. 

So it would be `SUPPORTS_GENERICS_ROUTINES` then.

Reply:
Yes, you are right, using the "routines" word would be cleaner. Can you push this change to this branch so that it appears in the merge request?

Reply:
Done. :)